### DISCUSSION TITLE
Best place to get help and/or discuss general ideas

### DISCUSSION THREAD

**Discussion by dolphins4all** [org-arl/UnderwaterAcoustics.jl#49] [General]
Is this the best place to discuss ideas or get help with UnderwaterAcoustics.jl implementation?   These wouldn't be bugs/issues...just getting help/ideas with integration into a larger model.

Or would it be more productive to discuss on Discourse, Slack, etc....?

Thanks! - Robert

**Comment by mchitre** [comment]
This is a fine place for such discussions

**Comment by dolphins4all** [comment]
Thanks.  I'll post the idea in a separate subject heading to avoid confusion
